src/opl/measurer.cpp

```diff
@@ -290,6 +360,13 @@ static void MeasureDurations(FmBank::Instrument *in_p, OPLChipBase *chip)
             break;
     }
 
+    if(false)
```

Comment:
You better define the macro `DEBUG_PLOT_OUTPUT` than producing a dead code :wink:

Reply:
Compiler takes care of it.
It shuts up a warning about the unused plotting function.

Reply:
Anyway, it's weird to have this, I prefer to use macro than dead code, it gives better visibility

Reply:
I let you try this at your home later and rewrite it as you like it.
As you can guess, all this is probably going to change results of measurements quite radically.
So there should be a decent amount of testing done before calling this a finished job.
Also, it's possible some hackish looking code around is going to lose purpose after this and can be eliminated.